mswindows/osgeo4w/package.sh

```diff
@@ -113,11 +113,7 @@ read PATCH <&3
 export VERSION=${MAJOR}.${MINOR}.${PATCH}
 export POSTFIX=${MAJOR}${MINOR}
 
-if [[ "$PATCH" == *svn* ]] ; then
-    GRASS_EXECUTABLE=grass${MAJOR}${MINOR}svn
-else
-    GRASS_EXECUTABLE=grass${MAJOR}${MINOR}
-fi
+GRASS_EXECUTABLE=grass${MAJOR}${MINOR}
```

Comment:
this seems not to be related to _OSGeo4W BAT files review_ ? please sort it out from this PR.

Reply:
See #445

Reply:
See #445